### PR TITLE
small fix

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

File: Document/0x05b-Basic-Security_Testing.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ $ cd apkx
 $ sudo ./install.sh
 ```
 
-This should copy `apkx` to `/usr/local/bin`. See [Decompiling Java Code](0x05c-Reverse-Engineering-And-Tampering.md#decompiling-java-code "Decompiling Java Code") for more information about usage.
+This should copy `apkx` to `/usr/local/bin`. See [Decompiling Java Code](0x05c-Reverse-Engineering-and-Tampering.md#decompiling-java-code "Decompiling Java Code") for more information about usage.
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md]: there are some more, search for "](0x05c-" ;)

Comment (cpholguera) [reply Document/0x05b-Basic-Security_Testing.md]: there should be at least 4 pieces in 5b

**Comment by OMTGreleaser** [comment]
Results for commit c7c55387b848e03743aa5a68681122e570d3d55e: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 2fdb58ed699b57fbfec5635af8cdeb07bb8fe1fa: Broken link result: [!] Error(s) found by the Links validator: 4 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by cpholguera** [comment]
Yet to be fixed:

- 0x05b-Basic-Security_Testing.md
```
 449,120: - If your device is not rooted, you can also use Frida, please refer to section [Dynamic Analysis on Non-Rooted Devices](0x05c-Reverse-Engineering-And-Tampering.md#dynamic-analysis-on-non-rooted-devices "Dynamic Analysis on Non-Rooted Devices").
```

**Comment by cpholguera** [comment]
And this one:

Note: Instant apps require an app-bundle. App-bundles are described in the [App Bundles](0x05a-platform-overview.md#app-bundles) section.

**Comment by OMTGreleaser** [comment]
Results for commit 1c6bdc524f8103316f02c511a2f1e8984f6fc618: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by commjoen** [comment]
> App Bundles
This is due to referring to it in a pr right?

**Comment by cpholguera** [comment]
That one's still in lowercase, sorry: `... described in the [App Bundles](0x05a-platform-overview.md#app-bundles) section.`

**Pull request merged**